src/Service/S3/tests/Integration/S3ClientTest.php

```diff
@@ -549,6 +557,60 @@ public function testListParts(): void
         self::assertSame('changeIt', $result->getRequestCharged());
     }
 
+    public function testPutBucketNotificationConfiguration(): void
+    {
+        $client = $this->getClient();
+
+        $input = new PutBucketNotificationConfigurationRequest([
+            'Bucket' => 'change me',
+            'NotificationConfiguration' => new NotificationConfiguration([
+                'TopicConfigurations' => [new TopicConfiguration([
+                    'Id' => 'change me',
+                    'TopicArn' => 'change me',
+                    'Events' => ['s3:ObjectCreated:*'],
+                    'Filter' => new NotificationConfigurationFilter([
+                        'Key' => new S3KeyFilter([
+                            'FilterRules' => [new FilterRule([
+                                'Name' => 'prefix',
+                                'Value' => 'change me',
+                            ])],
+                        ]),
+                    ]),
+                ])],
+                'QueueConfigurations' => [new QueueConfiguration([
+                    'Id' => 'change me',
+                    'QueueArn' => 'change me',
+                    'Events' => ['s3:ObjectCreated:*'],
+                    'Filter' => new NotificationConfigurationFilter([
+                        'Key' => new S3KeyFilter([
+                            'FilterRules' => [new FilterRule([
+                                'Name' => 'suffix',
+                                'Value' => 'change me',
+                            ])],
+                        ]),
+                    ]),
+                ])],
+                'LambdaFunctionConfigurations' => [new LambdaFunctionConfiguration([
+                    'Id' => 'change me',
+                    'LambdaFunctionArn' => 'change me',
+                    'Events' => ['s3:ObjectCreated:*'],
+                    'Filter' => new NotificationConfigurationFilter([
+                        'Key' => new S3KeyFilter([
+                            'FilterRules' => [new FilterRule([
+                                'Name' => 'suffix',
+                                'Value' => 'change me',
+                            ])],
+                        ]),
+                    ]),
+                ])],
+            ]),
+            'ExpectedBucketOwner' => 'change me',
+        ]);
+        $result = $client->PutBucketNotificationConfiguration($input);
+
+        self::assertTrue($result->resolve());
```

Comment:
If the fake S3 docker image support this feature, please add more assertions ont the reponse

I suggest to replace "change me" values by more friedly names

Reply:
What I can see the response is just a simple 200 status code

Reply:
Oh, indeed, I thought AWS returned an object.
It's ok then.